sycl/test-e2e/Graph/README.md

```diff
@@ -27,9 +27,35 @@ directories. These tests also define `GRAPH_E2E_EXPLICIT`
 or `GRAPH_E2E_RECORD_REPLAY` respectively to choose which API is used by the
 common code.
 
-The other directories are used to group similar tests together. They might
-themselves contain subdirectories named `Explicit` and `RecordReplay` if they
-make use of the framework described above.
+The other directories are used to group similar tests together. Tests that
+require a specific `aspect` are also grouped together in order to use the
+`lit.local.cfg` file. Directories might themselves contain subdirectories named 
+`Explicit` and `RecordReplay` if they make use of the framework described above.
 
 In addition, in order to help identify specific tests, the matching files
 in `Explicit`, `RecordReplay` and `Inputs` folders should have identical names.
+
+## Test Execution
+
+Tests might be run multiple times using different options. The most commonly used
+options are:
+
+- `l0_leak_check`: `lit` directive which, on the `level-zero` backend, enables
```

Comment:
```suggestion
- `l0_leak_check`: `lit` substitution which, on the `level-zero` backend, enables
```

Nit but this is a substitution that is a shortcut for `env UR_L0_LEAKS_DEBUG=1 SYCL_ENABLE_DEFAULT_CONTEXTS=0`, directives in `lit` terms I think only really applies to things like `XFAIL`, `REQUIRES` etc.